app/views/necesidad/solicitud_necesidad.html

```diff
@@ -246,12 +246,11 @@ <h1 class="page-header text-center">{{"SOLICITUD_NECESIDAD" | translate}} <span
                 <div class="col-md-6" ng-show="solicitudNecesidad.fields.General.PlanAnualAdquisiciones">
                   <label class="control-label">{{"PLAN_ANUAL_ADQUISICIONES" | translate}}</label>
                   <small>*{{"OBLIGATORIO" | translate}}</small>
-                  <select class="form-control" ng-model="solicitudNecesidad.vigencia"
+                  <select class="form-control" ng-model="solicitudNecesidad.planadquisicion"
                     ng-options="pa.descripcion for pa in solicitudNecesidad.planes_anuales track by pa.id"
                     required
                     ng-disabled="disabledSelect">
-                  <option value="">{{"SELECCIONAR" | translate}} {{"PLAN_ANUAL_ADQUISICIONES" |
-                                  translate}}</option>
+                  <option value="">{{"SELECCIONAR" | translate}} {{"PLAN_ANUAL_ADQUISICIONES" | translate}}</option>
```

Comment:
En general, para configurar una opcion como placeholder, se le pueden agregar los atributos `disabled` y `selected` a la vez - Probar lo siguiente:
```suggestion
                  <option disabled selected value="">{{"SELECCIONAR" | translate}} {{"PLAN_ANUAL_ADQUISICIONES" | translate}}</option>
```